README.md

```diff
@@ -225,6 +225,8 @@ For documentation on the latest development code see the [documentation index][d
 * [nginx_upstream_check](./plugins/inputs/nginx_upstream_check)
 * [nginx_vts](./plugins/inputs/nginx_vts)
 * [nsq_consumer](./plugins/inputs/nsq_consumer)
+* [nginx_vts](./plugins/inputs/nginx_vts)
```

Comment:
Remove this addition

Reply:
I'm confused, why would I need to remove the nginx_vts line?